.github/workflows/release.yml

```diff
@@ -14,37 +10,52 @@ on:
 env:
   GH_ANNOTATION: true
 jobs:
-  # todo: Keep in sync with `cloud_integration.yml`
   docker_build:
-    name: Docker build
     runs-on: ubuntu-18.04
+    strategy:
+      matrix:
+        target: [proxy, controller, web, cni-plugin, debug, cli-bin, grafana]
+    name: Docker build (${{ matrix.target }})
     steps:
     - name: Checkout code
       # actions/checkout@v2
       uses: actions/checkout@722adc6
-    - name: Setup SSH config for Packet
+    - name: Set environment variables from scripts
       run: |
-        mkdir -p ~/.ssh/
-        touch ~/.ssh/id && chmod 600 ~/.ssh/id
-        echo "${{ secrets.DOCKER_SSH_CONFIG }}"  > ~/.ssh/config
-        echo "${{ secrets.DOCKER_PRIVATE_KEY }}" > ~/.ssh/id
-        echo "${{ secrets.DOCKER_KNOWN_HOSTS }}" > ~/.ssh/known_hosts
-        ssh linkerd-docker docker version
+        . bin/_tag.sh
+        echo ::set-env name=TAG::$(CI_FORCE_CLEAN=1 bin/root-tag)
+
+        . bin/_docker.sh
+        echo ::set-env name=DOCKER_REGISTRY::$DOCKER_REGISTRY
+        echo ::set-env name=DOCKER_BUILDKIT_CACHE::${{ runner.temp }}/.buildx-cache
+    - name: Cache docker layers
+      # actions/cache@v2.0.0
+      uses: actions/cache@b820478
+      with:
+        path: ${{ env.DOCKER_BUILDKIT_CACHE }}
+        key: ${{ runner.os }}-buildx-${{ matrix.target }}-${{ env.TAG }}
+        restore-keys: |
+          ${{ runner.os }}-buildx-${{ matrix.target }}-
     - name: Build docker images
       env:
-        DOCKER_HOST: ssh://linkerd-docker
         DOCKER_TRACE: 1
       run: |
         export PATH="`pwd`/bin:$PATH"
-        bin/docker-build
-    - name: Create artifact with Windows CLI binary
+        docker buildx create --driver docker-container --use
```

Comment:
what about using existed actions to setup the buildx https://github.com/crazy-max/ghaction-docker-buildx
the action is set up with builtin capability to perform multi-arch build (Ref: https://github.com/crazy-max/ghaction-docker-buildx/blob/master/src/main.ts#L27)

Reply:
You definitely should make use of that action when you tackle multi-arch builds. For this PR, it's more practical to just use this command.